### DISCUSSION TITLE
Side-by-side Bible Text Display AKA Tabs-as-Columns

### DISCUSSION THREAD

**Discussion by ormris** [ezra-bible-app/ezra-bible-app#1145] [Ideas]
I generally like to read with some context shown when comparing multiple translations and passages. Showing tabs side-by-side would make this comparison more natural as all of the text would be viewable. There are a few caveats, though. This is probably only practical on a desktop or tablet display (vertical columns on mobile??). Columns would have focus, just like tabs, so there would be visual indication of what column is being effected by the book selector, for example. I think would probably be simpler than moving the buttons around to make it more clear what effects tabs vs the entire app. This layout would be selected in the settings page, rather than allowing drag-and-drop. I've photoshoped a mockup below. Basically what you would expect.
![ezra-columns](https://github.com/user-attachments/assets/d5d66bf1-d080-489d-8a03-548a99ff9a23)

**Comment by tobias-klein** [comment]
@ormris Thank you for your proposal and the time you spent on this. I have been considering to add a parallel translation mode since a while, see #30. I would plan to add a second column with the other verse text. Overall, this feature is quite some work and I need to see when I find the time to work on this. Potentially in the release that is coming up after the current one.

I agree that this is probably limited to desktop. But maybe it would work on bigger tablets as well in landscape mode? I have to see.

**Reply by ormris** [reply]
I'm hoping to spend some time on it as well. I think this would work at minimum on a landscape 7" tablet (slightly larger than a kindle). Smaller than that and I feel like the screen height wouldn't allow you to see much context. As for the second column view, I'd like to explore changing the "root" verse and the translation. Especially when I'm looking at cross-references, it's useful to be able to see both. For example, looking at a new testament quote of an old testament verse, and looking at the old testament verse at the same time.

**Comment by tobias-klein** [comment]
@ormris I started working on this now. See #1161.

**Reply by ormris** [reply]
Cool. Appreciate it!
